CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/results/AnalyticResult.java

```diff
@@ -82,14 +82,14 @@ public final int size() {
     public final void sort() {
 
         final List<Entry<IdentificationData, D>> mapPairs = new ArrayList<>(result.entrySet());
-        final List<D> mapValues = new ArrayList<>(result.values());
+        final List<AnalyticData> mapValues = new ArrayList<>(result.values());
```

Comment:
I would've thought that you would still want this (and the other couple that were changed) to be type `D` rather than `AnalyticData` since that is the type of the original result